examples/kubernetes-ingress/scripts/02-install-kubernetes-master.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
Add logs using tests/helpers.bash:log so users can follow what's going on in case they hit an issue while the script runs.